invenio.cfg

```diff
@@ -336,6 +347,13 @@ RDM_CUSTOM_FIELDS = [
     KeywordCF(name='openbiodiv:TaxonomicConceptLabel', multiple=True),
     # obo
     RelationshipListCF(name='obo:RO_0002453'),
+    # notes
+    *NOTES_CUSTOM_FIELDS
+]
+
+RDM_CUSTOM_FIELDS_UI = [
+    # notes CF
+    NOTES_CUSTOM_FIELDS_UI
```

Comment:
I have the same comment regarding naming in all the CF PRs. However, the rest of the content is good. Maybe:
a) open an issue about it
b) raise it in standup to discuss after

I think we could merge all (or most) PRs related to this. Nice job man!